x-pack/platform/plugins/shared/alerting/server/alerts_service/lib/create_concrete_write_index.ts

```diff
@@ -66,6 +68,7 @@ const updateUnderlyingMapping = async ({
   logger,
   esClient,
   concreteIndexInfo,
+  attempt = 1,
```

Comment:
should we limit the number of attempts? We can set it high, just to avoid an infinite recursion scenario.

Reply:
limited to 100